src/dbghelp.rs

```diff
@@ -409,8 +409,7 @@ pub fn init() -> Result<Init, ()> {
         //
         // See https://learn.microsoft.com/cpp/build/reference/pdbpath for an
         // example of where symbols are usually searched for.
-        let mut search_path_buf = Vec::new();
-        search_path_buf.resize(1024, 0);
+        let mut search_path_buf = vec![0; 1024];
```

Comment:
Hmm. This *could* be `with_capacity`, and then we pass the `search_path_buf.capacity()` when indicating how much space we have to allocate, and then we pass a `search_path_buf.set_len` after filling it.

...**or** that could be over-optimization as we are likely to get a fresh raw page from the OS because this was a calloc.

Reply:
fresh pages from the OS may still be more expensive than a recycled allocation from a pooling allocator. But I'd only play those kinds of games if this shows up in a benchmark and not just clippy linting about it.